core/src/main/java/com/github/kaktushose/jda/commands/dispatching/handling/command/SlashCommandHandler.java

```
@@ -78,7 +79,7 @@ protected InvocationContext<SlashCommandInteractionEvent> prepare(SlashCommandIn
         parsedArguments.addFirst(new CommandEvent(event, registry, runtime, command, replyConfig, dispatchingContext.embeds()));
 
         if (optionMappings.size() != optionDataDefinitions.size()) {
-            throw new IllegalStateException(
+            throw new JDACException.Internal(
                     "Command input doesn't match command options length! Please report this error to the devs of jda-commands."
```

"Please report this error to the devs of jda-commands." can be removed, would be duplicated